articles/custom_storage_location.md

```diff
@@ -222,6 +222,115 @@ f <- synStore(f)
 
 Please see the [REST docs](http://docs.synapse.org/rest/org/sagebionetworks/repo/model/project/ExternalS3StorageLocationSetting.html){:target="_blank"} for more information on setting external storage location settings using our REST API.
 
+## Setting Up an External Google Cloud Storage Bucket
+
+Follow the documentation on Google Cloud's site to **[Create a Bucket](https://cloud.google.com/storage/docs/creating-buckets){:target="_blank"}**.
+
+Make the following adjustments to customize it to work with Synapse:  
+
+* Select the newly created bucket and click the **Permissions** tab.
+    * Select the **Add members** button and enter the member `synapse-svc-prod@uplifted-crow-246820.iam.gserviceaccount.com`. This is Synapse's service account. Give the account the permissions "Storage Legacy Bucket Reader" and "Storage Object Viewer" for read permission. To allow Synapse to upload files, additionally grant the "Storage Legacy Bucket Writer" permission.
+
+
+<br/>
+
+For **read-write** permissions, you also need to create an object that proves to the Synapse service that you own this bucket. This can be done by creating an **<a href="/assets/downloads/owner.txt" download="owner.txt">owner.txt</a>** file with your Synapse username and uploading it to your bucket. You can upload the file with the Google Cloud Platform Console, or using the [gsutil application](https://cloud.google.com/storage/docs/gsutil){:target="_blank"}, you can upload using the command line. 
+
+<img id="imageSmall" src="/assets/images/ownerTxt.png">
+
+##### Command line
+
+```bash
+# copy your owner.txt file to your s3 bucket
+gsutil cp owner.txt gs://nameofmybucket/nameofmyfolder
+```
+
+##### Web
+
+Navigate to your bucket on the Amazon Console and select the **Upload files** button to upload your text file.
```

Comment:
Should "Amazon Console" be "Google Cloud Platform Console"?

Reply:
```suggestion
Navigate to your bucket on the Google Cloud Console and select the **Upload files** button to upload your text file.
```